server/test/commandHandlers/deployment/DeployServiceTest.js

```diff
@@ -84,6 +89,19 @@ describe('DeployService', function () {
       updateStatus: sinon.stub(),
       started: sinon.stub().returns(Promise.resolve({}))
     };
+    autoScalingTemplatesProvider = {
+      get: sinon.stub().returns(Promise.resolve([{
+        autoScalingGroupName: 'Name'
+      }]))
+    };
+    environmentTable = {
+      getByKey: sinon.stub().returns(Promise.resolve(''))
+    };
+    DynamoHelper = sinon.stub().returns(environmentTable);
+    DynamoHelperLoader = {
+      load: sinon.stub().returns(DynamoHelper)
+    };
+    getAsg = sinon.stub().returns(Promise.resolve(''));
 
```

Comment:
Have we added a test to ensure Deploy Service throws if an ASG or Environment is scheduled off?